packages/lab/src/progress/CircularProgress/CircularProgress.tsx

```diff
@@ -190,42 +147,38 @@ export const CircularProgress = forwardRef<
   return (
     <div
       className={clsx(className, "saltCircularProgress", {
-        [withBaseName("small")]: size === "small",
-        [withBaseName("medium")]: size === "medium",
-        [withBaseName("large")]: size === "large",
         [withBaseName("disabled")]: disabled,
       })}
```

Comment:
should change to
`className={clsx(withBaseName(), {
        [withBaseName("disabled")]: disabled,
      }), className}`
    
changes:  
1. className always comes last to make sure users can override our styles (see dev practises doc)
2. we use withBaseName, which has been defined above